### PR TITLE
Offer to Co-Maintain

### PR DESCRIPTION
**Pull request by ax3l**
Hi Jakob @wjakob and Sylvain @SylvainCorlay,

if you like, I hereby volunteer to co-maintain this feedstock :)

Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ source:
     sha256: {{ sha256 }}
 
 build:
-    number: 1001
+    number: 1002
```
Comment (SylvainCorlay) [inline recipe/meta.yaml]: No need to bump the build to add a feedstock maintainer

**Comment by SylvainCorlay** [comment]
Hey @ax3l I am happy to add you as a maintainer to the feestock.

I left a comment in your PR.

**Comment by ax3l** [comment]
@SylvainCorlay alright, updated accordingly :)

**Comment by wjakob** [comment]
I'd also be happy to have you as co-maintainer.

**Comment by ax3l** [comment]
Cool, thanks! :)

Feel free to merge the PR then, which will automatically add me.

**Pull request merged**